benefits/core/templates/core/agency_index.html

```diff
@@ -7,7 +7,7 @@
 {% block container_content %}
   <h1>{{ page.headline }}</h1>
 
-  <p>{% translate "core.pages.agency_index.h2" %}</p>
+  <h2>{% translate "core.pages.agency_index.h2" %}</h2>
```

Comment:
I'm a little confused on the flip-flopping of this, but I think I'll ignore it (?)

Reply:
https://github.com/cal-itp/benefits/pull/1020#discussion_r988393561